### PR TITLE
rsschool-2019Q1-cv.io

### PR DESCRIPTION
**Pull request by EvgenPrushk**
added information:
Contacts, Summary, Skills, Business qualities, Education,  Experience, English

### CODE REVIEW

**Review by arsafab — COMMENTED** [review]
- the repo name should be "rsschool-2019Q1-cv"
- html page should be available via the link "https://EvgenPrushk.github.io/rsschool-2019Q1-cv/"
- there are some unnecessary files in the project
- there are html validation issues
- not using editorconfig, html and css linting - this causes to style and semantic problems
- not all rules are followed from html academy style guide, like class names or importing styles
- code is semantic, it's nice but in some cases it was possible to use some html5 tags 
- commits follow style guide, cool))
To summarize:
Good job! If you will follow all instructions this can be perfect. Be more attentive to the style guides

File: index.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
```
Comment (arsafab) [inline index.html:10]: unnecessary line break

File: index.html [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header class="header">
+      <div class="resume">
+        <h1 class="resume__text">Resume</h1>
```
Comment (arsafab) [inline index.html:14]: class name should be like resume-text. NOT kebab naming or something like this

File: index.html [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+      </div>
+
+      <div class="header photo">
+        <img
+          class="photo__img"
+          src="./img/MyPhoto.png"
+          alt="MyPhoto"
+          width="300px"
+          height="300px"
+        />
```
Comment (arsafab) [inline index.html:24]: all attributes of the image tag shoul be in one line. These line breaks are validation error

File: index.html [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+          <div class="info__text">Prushak</div>
+          <div class="info__text">Evgen</div>
+          <div class="info__text">Vladimirovish</div>
+          <div class="info__text">Javascript Developer</div>
+        </div>
+      </div>
+    </header>
+
+    <section class="scills">
+      <h3 class="scills__text">Objective & Summary</h3>
```
Comment (arsafab) [inline index.html:37]: I think you mean class name 'skills'

File: index.html [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+        <li class="list__item">
+          Programming languages: Java Script – basics
+        </li>
+        <li class="list__item">Development Tools: Visual Studio Code</li>
+        <li class="list__item">
+          Web Technologies: HTML\СSS\Java Script – basics
+        </li>
+        <li>Defect & bug discovery</li>
+      </ul>
+
```
Comment (arsafab) [inline index.html:56]: Why do you use some li in one line and for another you use line breaks? it's difficult to review the code above

File: index.html [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+          Vavilov (Minsk), optomechanical and optoelectronic equipment working
+          Chief engineer
+        </li>
+        <li class="list__item">
+          <a class="list__link" href="https://github.com/EvgenPrushk"
+            >2018 Learning rolling-scopes-school (github.com/EvgenPrushk)
+          </a>
+        </li>
+      </ol>
+
```
Comment (arsafab) [inline index.html:111]: do not use line breaks for html text. this can lead to some problems in different browsers

File: index.html [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+                        </div>
+                    </div>
+                    <div class="footer__item">
+                        <div class="small-contact"><i class="small-contact__icon fas fa-map-marker-alt"></i><span class="small-contact__text">Есенина,
+                                39-204</span>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            </footer>
```
Comment (arsafab) [inline index.html:142]: styling of the footer block is not so good

File: style.css [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+    -webkit-justify-content: space-between;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    padding: 41px 7%;
+}
+
```
Comment (arsafab) [inline style.css:105]: there is no need to use here vendor prefix because you focus browser is the lates chrome

File: style.css [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+.footer__content {
+    display: -webkit-box;
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: justify;
+    -webkit-justify-content: space-between;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+    padding: 41px 7%;
```
Comment (arsafab) [inline style.css:103]: not combine absolute and relative units in one css property

File: style.css [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+    color: #FF5E5E;
+    margin-right: 10px;
+    font-size: 18px;
+}
+
+.small-contact__text {
+    font-size: 14px;
+    line-height: 16px;
+    color: black;
+}
```
Comment (arsafab) [inline style.css:118]: no empty line. .editorconfig could fix this problem automatically

File: style.css [hunk 11]
```diff
@@ -0,0 +1,118 @@
+body {
```
Comment (arsafab) [inline style.css:1]: for all file. You need to use properties according to the rule about the order of css properties. Firstly about positioning, then about box model etc.

then why do you use width 1024 px for every block? it would be more rational to wrap all content in one block and center it
